src/config/section/network.js

```diff
@@ -29,7 +29,16 @@ export default {
       icon: 'apartment',
       permission: ['listNetworks'],
       resourceType: 'Network',
-      columns: ['name', 'state', 'type', 'cidr', 'ip6cidr', 'broadcasturi', 'account', 'zonename'],
+      columns: ['name', 'state',
+        {
+          type: (record) => {
+            if (record.vpcid && record.vpcid !== '') {
+              return 'VPC Tier'
```

Comment:
This is new, all VPC tiers are isolated networks - should this be changed in backend? I worry if it would confuse users.

Reply:
@rhtyd this is just a string displaying that it belongs to vpc..I dont think we should change anything in background.
I have lot of networks then we can use this string to quickly identify which ones belong to vpc

Reply:
Should we add a VPC column then, keep the type as-is, but by introducing a `vpcid` column it will make it clicky too.

Reply:
**+1** on having such information on UI as proposed by @rhtyd.
What about having the VPC name provided instead of just pointing it as a _VPC Tier_? Or do you think that it would be too much for this window?